spec/CloudCode.spec.js

```diff
@@ -1470,4 +1470,36 @@ describe('afterFind hooks', () => {
           });
       });
   });
+
+  it('should report count if passed', (done) => {
+    const hook = {
+      method: function(req) {
+        expect(req.count).toBe(true);
+        return Promise.resolve();
+      }
+    };
+    spyOn(hook, 'method').and.callThrough();
+    Parse.Cloud.beforeFind('Stuff', hook.method);
+    new Parse.Query('Stuff').count().then((count) => {
+      expect(count).toBe(0);
+      expect(hook.method).toHaveBeenCalled();
+      done();
+    });
+  });
+
+  it('should report count if passed', (done) => {
```

Comment:
the test description is the same as above.  maybe should be 'should report find if passed', but i'm confused what you're testing and why.  what does it mean "to report" in this context?

i just see that the result is zero as expected and your hook is called for both find and count.